wpilibj/src/main/java/edu/wpi/first/wpilibj/controller/PIDController.java

```diff
@@ -74,13 +74,16 @@ public PIDController(double kp, double ki, double kd) {
    * @param kp The proportional coefficient.
    * @param ki The integral coefficient.
    * @param kd The derivative coefficient.
-   * @param period The period between controller updates in seconds.
+   * @param period The period between controller updates in seconds. Must be non-zero and positive.
```

Comment:
We should add this note to the C++ docs; even though we probably don't want an exception, emphasizing it in the Doxygen is a good idea.